src/components/UriViewer.js

```diff
@@ -5,13 +5,15 @@ import { div, h, img, input } from 'react-hyperscript-helpers'
 import { requesterPaysWrapper, withRequesterPaysHandler } from 'src/components/bucket-utils'
 import Collapse from 'src/components/Collapse'
 import { ButtonPrimary, ClipboardButton, Link } from 'src/components/common'
+import { FileProvenance } from 'src/components/data/data-table-provenance'
 import { getDownloadCommand, getUserProjectForWorkspace, parseGsUri } from 'src/components/data/data-utils'
 import { spinner } from 'src/components/icons'
 import Modal from 'src/components/Modal'
 import DownloadPrices from 'src/data/download-prices'
 import { Ajax } from 'src/libs/ajax'
 import { bucketBrowserUrl } from 'src/libs/auth'
 import colors from 'src/libs/colors'
+import { isDataTableProvenanceEnabled } from 'src/libs/config'
```

Comment:
Out of curiosity, why put this behind a feature flag?

Reply:
This feature is kind of experimental, so we wanted to collect some feedback and maybe adjust before making it generally available.
https://broadworkbench.atlassian.net/browse/SU-180